pandas/tools/pivot.py

```diff
@@ -438,18 +448,97 @@ def crosstab(index, columns, values=None, rownames=None, colnames=None,
     data.update(zip(rownames, index))
     data.update(zip(colnames, columns))
 
+    if values is None and aggfunc is not None:
+        raise ValueError("aggfunc cannot be used without values.")
+
+    if values is not None and aggfunc is None:
+        raise ValueError("values cannot be used without an aggfunc.")
```

Comment:
Can you maybe also add a notice of this in the docstring? For example in the explanationo of `values`: "``Need to specify `agfunc` ``"